apps/spruce/src/components/TupleSelect/index.tsx

```diff
@@ -36,40 +45,36 @@ const TupleSelect: React.FC<TupleSelectProps> = ({
 
   return (
     <Container>
```

Comment:
I wish overflow typing wasn't covered by the ➕ icon 😢 no need to fix here though